app/brave_generated_resources.grd

```diff
@@ -311,7 +311,7 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Tor
       </message>
       <message name="IDS_LOCATION_BAR_OPEN_IN_TOR_TOOLTIP_TEXT" desc="The tooltip text of tor button in location bar.">
-        Open in Tor
+        Open in Tor<ph name="URL">$1</ph>
```

Comment:
Can leave out "open in Tor". or put "Open in Tor: <ph name="URL">$1</ph>".

Reply:
I also prefer just showing the URL without any message. This makes it similar to what is displayed in the bottom left corner of the browser when the mouse hovers over regular links in a page.

Reply:
Done.